### PR TITLE
naming-structure

### PR DESCRIPTION
**Pull request by vanessa-w**
Adding the dxw document naming convention

### CODE REVIEW

File: playbook.md [hunk 1]
```diff
@@ -899,6 +899,11 @@ We use Google Apps for Work to manage calendars, write and share documents. Ther
 
 When we write new things, we try to save them in a sensible folder within the existing structure.
 
+The dxw naming structure for documents is:
+
+Year-Month-Day    Doc type   Title
+E.g. 2017-01-01 Order Summary blog.gov.uk
```
Comment (dgmstuart) [inline playbook.md]: @vanessa-w What should this date correspond to? The date it was originally created? The date it was sent? (if external)

**Comment by vanessa-w** [comment]
The naming structure is used in our internal and client documentation. 
It's printed and up in the kitchen and new starters are made aware of it.  
This was to avoid the frequent use of titles such as "Contract" or "Order Summary" which was making filing frustrating.
The individual will choose the most appropriate date - be it the date of a contract or the date they created a document. We were aiming for the guidance to be as light touch as possible, given the subject.

**Comment by dgmstuart** [comment]
@vanessa-w Thank you - that's a very clear explanation. If we could add some more words to this section to reflect that this applies to internal and client docs then we can get this merged in.

**Comment by dgmstuart** [comment]
Superseded by https://github.com/dxw/playbook/pull/103

**Pull request closed without merging**